pkg/tagger/collectors/ecs_fargate_extract.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build docker
+
+package collectors
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/tagger/utils"
+	"github.com/DataDog/datadog-agent/pkg/util/docker"
+	"github.com/DataDog/datadog-agent/pkg/util/ecs"
+)
+
+// ignore these container labels as we already have them in task metadata
+var isBlacklisted = map[string]bool{
+	"com.amazonaws.ecs.cluster":                 true,
+	"com.amazonaws.ecs.container-name":          true,
+	"com.amazonaws.ecs.task-arn":                true,
+	"com.amazonaws.ecs.task-definition-family":  true,
+	"com.amazonaws.ecs.task-definition-version": true,
+}
+
+// parseMetadata parses the the task metadata, and its container list, and returns a list of TagInfo for the new ones.
+// It also updates the lastSeen cache of the ECSFargateCollector and return the list of dead containers to be expired.
+func (c *ECSFargateCollector) parseMetadata(meta ecs.TaskMetadata) ([]*TagInfo, []string, error) {
+	var output []*TagInfo
+	seen := make(map[string]interface{}, len(meta.Containers))
+
+	if meta.KnownStatus != "RUNNING" {
+		return output, nil, fmt.Errorf("Task %s is in %s status, skipping", meta.Family, meta.KnownStatus)
+	}
+
+	for _, ctr := range meta.Containers {
+		seen[ctr.DockerID] = nil
+		if _, found := c.lastSeen[ctr.DockerID]; !found {
+			tags := utils.NewTagList()
+
+			// cluster
+			tags.AddLow("ecs_cluster_name", meta.ClusterName)
+
+			// task
+			tags.AddLow("ecs_task_family", meta.Family)
```

Comment:
Hi! We're looking forward to using these tagger additions in the process agent :)

At a quick glance, I've noticed that these tag names are inconsistent with those from the other extractors; is there any way we can change them to match whats in `{docker,ecs}_extract.go`?

e.g:
`"ecs_task_family" -> "task_name"`, 
`"ecs_task_version" -> "task_version"`, 
`"imageName" -> "image_name"`, etc.